specs/tilebag_spec.rb

```diff
@@ -0,0 +1,86 @@
+require_relative 'spec_helper'
+
+describe "Tilebag" do
+
+  before do
+
+    @test_ob = Scrabble::Tilebag.new
+
+    default_tiles = {
+      A: 9,
+      B: 2,
+      C: 2,
```

Comment:
Re-implementing this hash here is pretty redundant, and is subject to the same problems that you had putting it in `TileBag` in the first place, especially since you probably just copy-pasted it. Instead, it's best to trust that you got the numbers right, and just check that, for example, the total number of tiles is correct.